k6/quicksilver.js

```diff
@@ -31,7 +70,7 @@ export const options = {
       executor: 'constant-vus',
       vus: 3,
       duration: '20s',
-      startTime: '22s'
+      startTime: '66s'
```

Comment:
What's going on with these startTimes?  Staggering the tests?

Reply:
Yes, exactly. See https://community.grafana.com/t/sequentially-run-scenarios/99153/3 for some context